src/componmnet/HeatMap/index.jsx

```diff
@@ -2,19 +2,18 @@ import React, { Component, useEffect, useRef, useState } from 'react';
 import cgl from '../../lib/clustergrammer-gl.node';
 import data from '../../../data/cytof.json';
 
-function HeatMap({setNode}) {
+function HeatMap({setNode, heatMapStyle}) {
```

Comment:
The parameter should be `style`
```suggestion
function HeatMap({setNode, style}) {

```